docs/install-infra-on-k0s.md

```diff
@@ -0,0 +1,168 @@
+  * [OpenEBS](https://docs.k0sproject.io/head/storage/)
+
+This guide assumes that you've already installed the Infra CLI. You
+can check out the [Quickstart guide](https://infrahq.com/docs/getting-started/quickstart)
+which contains directions on how to install the CLI.
+
+### 1. Download k0s
+
+`curl -sSLf https://get.k0s.sh | sudo sh`
+
+1. Create a k0s config file
+
+As the root user, run:
+
+`k0s config create > /etc/k0s/k0s.yaml`
+
+This will dump out a k0s config file which you will need to edit.
+
+### 2. Edit /etc/k0s/k0s.yaml to include OpenEBS
+
+Modify `/etc/k0s/k0s.yaml` and include the section:
+
+```
+spec:
+  extensions:
+    storage:
+      type: openebs_local_storage
+```
+
+### 3. Install k0s
+
+Run the command:
+
+`sudo k0s install controller --single -c /etc/k0s/k0s.yaml`
+
+If this is a multinode system, you can omit `--single` flag when you're doing the install, otherwise
+the controller and the kubernetes node will be created on the same instance. Once you've installed
+k0s, start it with the command:
+
+`sudo k0s start`
+
+### 4. Check that everything is running
+
+First check to make certain that k0s is up and running. It may take a few moments before
+everything is up.
+
+`sudo k0s status`
+
+Also, check to make certain that OpenEBS is working correctly. Run the command:
+
+`sudo k0s kubectl get storageclass`
+
+This will output something that looks like:
+```
+NAME               PROVISIONER        RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+openebs-device     openebs.io/local   Delete          WaitForFirstConsumer   false                  24s
+openebs-hostpath   openebs.io/local   Delete          WaitForFirstConsumer   false                  24s
+```
+
+
+### 5. Export a kubeconfig
+
+`sudo k0s kubectl view --raw`
+
+Save this into `~/.kube/config` (make sure you're not writing over an existing kube config) and you can now
+use `kubectl` directly. You'll need to do this to work with Infra.
+
+### 6. Install the Metallb load balancer
+
+```
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/metallb.yaml
+```
+
+### 7. Create a ConfigMap to set up your IP address pool
+
+Create the file `metallb-configmap.yaml` which looks something like:
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - <ip-address-range-start>-<ip-address-range-stop>
+```
+
+You can use private IP addresses in the IP Address range fields, but you'll need to make certain that any IP addresses are routed correctly.
+For example, if your network is using `192.168.1.1` to `192.168.1.254`, you can pick a small range of IPs inside of that network
+(make certain that your router isn't handing those IP addresses out with DHCP).
```

Comment:
How can I make check this?

Reply:
Added a note about `netstat -rn` and `route -n`.